test/Cache.js

```diff
@@ -123,6 +123,33 @@ test('if expireAfter will clear the timeout and add a new one when updateExpire
   t.is(cache.expirations.length, 0);
 });
 
+test('if onExpire callback can cancel expireAfter', async (t) => {
```

Comment:
so i just want to be clear because of the name of this test ... this will not actually cancel expiration, just reset it based on the `maxAge`. that is the intention yes? not to permanently opt out of expiring the cache?

Reply:
You are correct. Let me remove the confusion. Apologies, English is not my native tongue.
Makes me think `updateExpire` could be `updateAge` in the other proposal if that makes things more clear for other users, but it's merely cosmetics from the outside.

Reply:
"engines" : { "node" : ">=0.10.3 <0.12" }